### PR TITLE
Adds support for allowing backups of unlimited size

### PR DESCRIPTION
**Pull request by ajyey**
This MR adds support for allowing backups of unlimited size (denoted by 0).

Addresses #3157

### CODE REVIEW

File: client/pages/config/backups.vue [hunk 1]
```diff
@@ -64,7 +64,7 @@
         <ui-text-input type="number" v-model="maxBackupSize" no-spinner :disabled="updatingServerSettings" :padding-x="1" text-center class="w-10" @change="updateBackupsSettings" />
 
         <ui-tooltip :text="$strings.LabelBackupsMaxBackupSizeHelp">
-          <p class="pl-4 text-lg">{{ $strings.LabelBackupsMaxBackupSize }} <span class="material-symbols icon-text">info</span></p>
+          <p class="pl-4 text-lg">{{ $strings.LabelBackupsMaxBackupSize }} (0 for unlimited) <span class="material-symbols icon-text">info</span></p>
```
Comment (nichwall) [inline client/pages/config/backups.vue:67]: This should be added to the English string file in `client/strings` instead of adding a constant string value

Comment (ajyey) [reply client/pages/config/backups.vue]: Done 👍

Comment (advplyr) [reply client/pages/config/backups.vue]: @nichwall I wonder if changing that string in en-us will require the already translated strings in other languages to get reset

Comment (ajyey) [reply client/pages/config/backups.vue]: Was wondering that as well. That might be outside my wheelhouse to be honest 😓

**Review by nichwall — COMMENTED** [review]
Looks good to me. I put some random large files in `/metadata/items` (so my backups were about 7 GB) and everything is working as expected with both unlimited and maximum size set.
A new server did also correctly set the default maximum size to 1 GB.

**Comment by advplyr** [comment]
Thanks!

File: client/pages/config/backups.vue (hunk 1 above)
Comment (nichwall) [reply client/pages/config/backups.vue]: Just checked Weblate, the translations were automatically marked as needing updates when the English string was updated

**Comment by ZLoth** [comment]
Thank you!

**Pull request merged**